### PR TITLE
make series axis parameter docs consistent

### LINKED ISSUE #45969: DOC: Series.rolling and DataFrame.rolling share docstring
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/reference/api/pandas.Series.rolling.html?highlight=series%20rolling#pandas.Series.rolling

### Documentation problem

The documentation of the Series function contains references to axis, which makes no sense for a Series (which is one-dimensional).  It also refers to single vs. table functionality, which again makes no sense for a Series (which is single-only).

I can see that in cases like drop and dropna, axis exists as a parameter, but clearly states that there is only one axis.  In other documentation (such as groupby) the series documentation invites both row and column axis.

### Suggested fix for documentation

The Series documentation for rolling should follow the pattern of explaining axis as provided by drop/dropna.  All Series functions with an axis parameter should be checked and fixed if necessary.

### LINKED ISSUE #47100: BUG: Series.clip should not take axis argument
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the main branch of pandas.

### Reproducible Example

```python
import pandas as pd
pd.Series([1]).clip(3, axis=1)
```

### Issue Description

`Series.clip` takes an `axis` argument documented [here](https://pandas.pydata.org/docs/reference/api/pandas.Series.clip.html), but there's only one axis for a `Series` and passing `axis=1` always gives `ValueError: No axis named 1 for object type Series`.

### Expected Behavior

`axis` should be an invalid argument to `Series.clip`.

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : b4e578db857f0705144a56eac0ab65d6fb0a8760
python           : 3.9.12.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 21.3.0
Version          : Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.5.0.dev0+122.gb4e578db85.dirty
numpy            : 1.22.1
pytz             : 2021.3
dateutil         : 2.8.2
pip              : 22.0.4
setuptools       : 59.8.0
Cython           : 0.29.27
pytest           : 7.0.0
hypothesis       : None
sphinx           : 4.4.0
blosc            : None
feather          : None
xlsxwriter       : 3.0.3
lxml.etree       : 4.8.0
html5lib         : None
pymysql          : None
psycopg2         : 2.9.3
jinja2           : 3.0.3
IPython          : 8.0.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fastparquet      : None
fsspec           : 2022.3.0
gcsfs            : None
matplotlib       : 3.5.1
numba            : None
numexpr          : 2.8.1
odfpy            : None
openpyxl         : 3.0.9
pandas_gbq       : 0.16.0
pyarrow          : 6.0.1
pyreadstat       : None
pyxlsb           : None
s3fs             : 2022.3.0
scipy            : 1.7.3
sqlalchemy       : 1.4.31
tables           : 3.7.0
tabulate         : None
xarray           : 0.20.2
xlrd             : 2.0.1
xlwt             : None
zstandard        : None

</details>

### PR DESCRIPTION
**Pull request by jackgoldsmith4**
- [x] closes #47100
- [x] closes #45969 
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

### CODE REVIEW

**Comment by jackgoldsmith4** [comment]
@simonjayhawkins I'm having trouble debugging these test failures. Do you have an idea why they're failing?

**Comment by jackgoldsmith4** [comment]
The one check that's failing seems to be unrelated: https://github.com/codecov/codecov-action/issues/745

File: pandas/core/generic.py [hunk 1]
```diff
@@ -5086,8 +5090,8 @@ def filter(
             Keep labels from axis for which re.search(regex, label) == True.
         axis : {0 or ‘index’, 1 or ‘columns’, None}, default None
             The axis to filter on, expressed either as an index (int)
-            or axis name (str). By default this is the info axis,
-            'index' for Series, 'columns' for DataFrame.
+            or axis name (str). By default this is the info axis, 'columns' for
+            DataFrame.For `Series` this parameter is unused and defaults to None.
```
Comment (jreback) [inline pandas/core/generic.py:5094]: space here

Comment (jreback) [inline pandas/core/generic.py:5094]: can you use backticks for `None`

**Review by jreback — COMMENTED** [review]
minor comments

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @jackgoldsmith4

**Pull request merged**